arcgis-ios-sdk-samples/Display information/Query features with Arcade expression/QueryFeaturesArcadeExpressionViewController.swift

```diff
@@ -106,6 +108,8 @@ extension QueryFeaturesArcadeExpressionViewController: AGSGeoViewTouchDelegate {
             guard let results = results, let self = self else { return }
             // Get the selected feature.
             if let elements = results.first?.geoElements, let identifiedFeature = elements.first as? AGSArcGISFeature {
+                // Disable user interaction after a feature has been identified.
+                self.mapView.isUserInteractionEnabled = false
```

Comment:
Was there a specific issue this is trying to address?

If you're concerned about an "identify" operation happing at the same time as the arcade evaluation, you could move the block of code [here](https://github.com/Esri/arcgis-runtime-samples-ios/blob/fda1fb4edb0cf1d64dfa040052f7e0f0b577f415/arcgis-ios-sdk-samples/Display%20information/Query%20features%20with%20Arcade%20expression/QueryFeaturesArcadeExpressionViewController.swift#L64-L66) (in the `evaluateArcadeInCallout` method) to [here](https://github.com/Esri/arcgis-runtime-samples-ios/blob/fda1fb4edb0cf1d64dfa040052f7e0f0b577f415/arcgis-ios-sdk-samples/Display%20information/Query%20features%20with%20Arcade%20expression/QueryFeaturesArcadeExpressionViewController.swift#L106) (geoView(_ geoView: AGSGeoView, didTapAtScreenPoint)

That way, the arcade evaluation is canceled as soon as the user taps again on the map.

Reply:
I was trying to address this [issue](https://github.com/Esri/arcgis-runtime-samples-ios/pull/1182#pullrequestreview-1020937384). I thought I resolved it after canceling `evaluateOperation` but I tested it again and am experiencing the same issue. I tried out your suggestion but I'm still seeing the same bug but I realize it's similar logic to disabling user interaction on the map. Do you have any insight to this isse?

Reply:
It would be helpful to see if the trailing closure for `evaluator.evaluate` was called after a call to `evaluateOperation.cancel()`.  I'm not sure if it's supposed to be called, but it seems reasonable to think that it would be called with a "user cancelled" error.  It it's not it may be an error in the SDK.

Otherwise, you *could* put the call to `UIApplication.shared.hideProgressHUD()` after the call to `evaluateOperation.cancel()`.  That would indicate whether the closure was being called after cancel.

Reply:
@mhdostal Hmm, the closure for `evaluator.evaluate` is still called after canceling `evaluateOperation`. But isn't that expected since `evaluateOperation` only holds the previous `AGSCancelable` so it wouldn't affect the current evaluation?